package.json

```diff
@@ -7,40 +7,40 @@
     "url": "https://github.com/mozilla/chronicle/issues"
   },
   "dependencies": {
-    "bell": "2.0.0",
+    "bell": "2.1.0",
     "boom": "2.6.1",
     "commander": "2.6.0",
-    "convict": "0.6.0",
-    "elasticsearch": "3.1.1",
-    "embedly": "1.0.2",
-    "hapi": "8.1.0",
+    "convict": "0.6.1",
+    "elasticsearch": "3.1.3",
+    "embedly": "1.0.4",
+    "hapi": "8.2.0",
```

Comment:
hapi changelog 8.1.0 -> 8.2.0: https://github.com/hapijs/hapi/commits/v8.2.0

- rev inert from 2 to 2.1, update docs
  - inert changes from 2 to 2.1: allow directory listing to be a file other than `index.html`; test-related changes
- route errors return boom object, not just error message (2309)
- make Options response body empty, not empty object (2359)
- Don't respond to connections until listening is started (2362)
- Refuse to handle incoming request after server is stopped (2363)
- bump catbox dep (we don't use catbox)
- escape error messages returned by validation failure (prevents xss echo attack) (2370)
- bump inert dep from 2.1.0 to 2.1.2:
  - bugfix: return 403 if file can't be read (19)
  - bugfix: return 403 if path contains null bytes (19)
- bump mime-db from 1.5.0 to 1.6.1 https://github.com/jshttp/mime-db/blob/master/HISTORY.md (2395)
  - nothing here should impact us, it's just adding new mime types
- routes should allow null input if explicitly allowed (2392)
- throw if routes are attached before connection is available (otherwise you'll get strange 404s) (2397)
- fix bug with route pre- handler (we don't use this)